src/test/java/part1/exercise/StreamsExercise1.java

```diff
@@ -49,11 +76,13 @@ public void sumEpamDurations() {
             }
         }
 
-        // TODO
-        throw new UnsupportedOperationException();
-
-        // int result = ???
-        // assertEquals(expected, result);
+        int result = employees.stream()
+                .map(Employee::getJobHistory)
+                .flatMap(Collection::stream)
+                .filter(j -> employerWorkedForEpam.test(getEmployer.apply(j)))

```

Comment:
It's really less readable, but we already have employerWorkedForEpam function for compearing. 
I just wanted to use it